index.html

```diff
@@ -0,0 +1,54 @@
+
+<!DOCTYPE html>
+
+<html lang="en">
+
+  
+    <meta charset="UTF-8" />
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;1,200;1,300;1,400;1,500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="styles.css" />
+    <title>portfolio</title>
+  </head>
+  <body>
+      
+        <div class="container">
+        <div class="section">
+        
+      <div class="mainsection">
+        <div class="maincontainer">
+      <div class="section1">
+        <h2 class="text1">
```

Comment:
- [ ] Kindly make sure the header part of your portfolio should be in a `header` tag
- [ ] Kindly put the logo (if the design has one) is on the left and have a link (`a` tag)
- [ ] Kindly make sure the hamburger button is on the right and has a `nav` tag